packages/opds-web-client/src/components/__tests__/DownloadButton-test.tsx

```diff
@@ -101,18 +107,25 @@ describe("DownloadButton", () => {
     expect(fulfill.args[0][0]).to.equal("download url");
   });
 
-  it("opens indirect fulfillment link in new tab", (done) => {
+  it("opens indirect fulfillment link in new tab", done => {
```

Comment:
I remember that this was something that the linter complained about. Does prettier override those complaints?

Reply:
Yea the idea is that prettier will allow us to disable all lint rules that deal with formatting. So you'll never get complained at for that again, because it's formatted automatically. 

In terms of this specific formatting option, we can set it to either have the parens or not have them on single argument arrow functions, totally up to you which you prefer. No parens was just the default. It of course will always have parens when there are multiple arguments